pressbooks/pressbooks-book-directory-fe
# Add scroll action by clicking on collection cards

This change add a scroll action to the `#current-filters ` level by clicking in one of the collections cards at the header of the application. This is a new ID added at the level of the chips that represents the filters applied.

### How to test
- Set in `.env.local` file one of the indexes that contains `collections` fields populated in Algolia. At the moment of this PR the `dev_pressbooks_directory` Algolia's index contains this field.
- Click in any card in the collections section in the header (above of Search input).
- The page should be scrolled automatically at the top of the book cards, at the Sort By and Results level vertically.
